app/controllers/measurements_controller.rb

```diff
@@ -102,9 +102,11 @@ def destroy
   end
 
   def count
-    @count = {}
-    @count[:count] = Measurement.count
-    respond_with @count
+    @count = Rails.cache.fetch("measurement/#{Measurement.maximum(:id)}/count") do
```

Comment:
Is it okay to change `@count` to a `Integer`? It was a `Hash` originally.

Reply:
It depends on how you are serious about number. I would cache for a certain period like:

```ruby
Rails.cache.fetch('mesurement/count', expires_in: 5.minutes) { Measurement.count }
```

because `id` of `measuremnts` table might change a lot when someone is importing.

Reply:
If it is available, you might want to consider using `race_condition_ttl` option (https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-store) not to issue multiple`COUNT` queries in case of when many people are accessing the same page.

Reply:
> Is it okay to change `@count` to a `Integer`? It was a `Hash` originally.

Isn't `count` returning integer anyways? or you meant biting, or something else?

Reply:
> It depends on how you are serious about number. I would cache for a certain period like:
> 
> ```ruby
> Rails.cache.fetch('mesurement/count', expires_in: 5.minutes) { Measurement.count }
> ```
> 
> because `id` of `measuremnts` table might change a lot when someone is importing.

I think it should be fine without `expires_in`. This is what I meant, expire when someone uploads a new measurement. BTW @auspicacious, how often someone uploads a new measurement?

Reply:
> If it is available, you might want to consider using `race_condition_ttl` option (https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-store) not to issue multiple`COUNT` queries in case of when many people are accessing the same page.

I'm not sure how often someone accesses this endpoint. @matschaffer do you know how to check it in kibana?

Reply:
> Isn't count returning integer anyways? or you meant biting, or something else?

In [this view](https://github.com/Safecast/safecastapi/blob/master/app/views/measurements/count.html.erb), `@count` needs be Hash-like object.

Reply:
Should be fine, I [changed](https://github.com/Safecast/safecastapi/pull/678/files#diff-c255ce3e484bdc6d85dfeb4f27b92fe9R1) it. Also, added spec

Reply:
I see, missed that. Sorry.

Reply:
@sasharevzin Users upload "drives," which contain many measurements. This table might not change for a few hours, then it will suddenly increase by thousands of new rows.

As Eito said, "id of measuremnts table might change a lot when someone is importing."

Reply:
Right, it might change and it will change. That's why I added `max(:id)` to the key to making sure it has the latest `count`